docs/plugins/filters.asciidoc

```diff
@@ -56,6 +56,9 @@ include::filters/aggregate.asciidoc[]
 :edit_url: https://github.com/logstash-plugins/logstash-filter-alter/edit/master/docs/index.asciidoc
 include::filters/alter.asciidoc[]
 
+:edit_url: https://github.com/logstash-plugins/logstash-filter-anonymize/edit/master/docs/index.asciidoc
```

Comment:
Not sure why we are adding this back in. It's causing build errors because we removed the docs for this plugin in this PR: https://github.com/elastic/logstash-docs/pull/428